web/app/js/components/ResourceDetail.jsx

```diff
@@ -347,6 +365,18 @@ export class ResourceDetailBase extends React.Component {
           title="TCP"
           isTcpTable={true}
           metrics={this.state.podMetrics} />
+
+        {
+          noEdges ? null :
```

Comment:
if you're not using `noEdges` anywhere else, might as well just directly say `_isEmpty(this.state.edges)` here